src/pages/registration/resource_type_tab/sub_type_forms/JournalForm.tsx

```diff
@@ -47,7 +47,9 @@ export const JournalForm = () => {
         <JournalDetailsFields />
       </InputContainerBox>
 
-      {nviApplicableTypes.includes(instanceType) ? <NviValidation registration={values} /> : null}
+      {nviApplicableTypes.findIndex((category) => category === instanceType) >= 0 ? (
```

Comment:
kan komme unna med `includes` her også, om vi sjekker at det er en verdi for instance type først
```suggestion
      {instanceType && nviApplicableTypes.includes(instanceType) ? (
```